src/app.js

```diff
@@ -1,21 +1,34 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
+const url = require('url');
 
-module.exports = sum;
+const server = http.createServer((req, res) => {
+  res.setHeader('Content-Type', 'application/json');
+
+  const normalizedUrl = new url.URL(req.url, `http://${req.headers.host}`);
+
+  const parts = normalizedUrl.pathname.slice(1).split('/');
+  const query = {};
+
+  for (const [key, value] of normalizedUrl.searchParams.entries()) {
+    if (!query[key]) {
+      query[key] = value;
+    } else {
+      if (query[key] instanceof Array) {
+        query[key].push(value);
+      } else {
+        query[key] = [query[key], value];
+      }
+    }
+  }
+
+  res.end(
+    JSON.stringify({
+      parts,
+      query,
+    })
+  );
+});
+
+server.listen(3000);
```

Comment:
It's always a good idea to notify that server has started
```suggestion
server.listen(3000, () => console.log("Server started on 3000");
```